src/components/confirm/index.tsx

```diff
@@ -0,0 +1,65 @@
+import { Component, Event, EventEmitter, h, Host, Prop, State, VNode } from "@stencil/core"
+import { Color, Data, Fill } from "../../model"
+
+@Component({
+	tag: "smoothly-button-confirm",
+	styleUrl: "style.css",
+	scoped: true,
+})
+export class SmoothlyButtonConfirm {
+	timer?: NodeJS.Timeout
```

Comment:
This should be typed as number. use `window.setTimeout()` and `window.clearTimeout()` for correct type